index.js

```diff
@@ -47,31 +47,28 @@ function continueOrFail(failOnUnused, compilation, allFiles) {
   }
 }
 
-function display(filesByDirectory) {
+function display(compilation, filesByDirectory) {
+  const log = compilation.getLogger
+    ? msg => compilation.getLogger('UnusedPlugin').warn(msg)
+    : msg => process.stderr.write(chalk.red(`UnusedPlugin: ${msg}\n`));
```

Comment:
Chalk is now only used in fallback mode to print in console in red. 
Might be worth dropping.